### PR TITLE
Fix broken import in cloudformation

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY
* Fix imports
* Add new GPLv3 Licence

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
lib/ansible/modules/cloud/amazon/cloudformation.py

##### ANSIBLE VERSION

```
2.4devel
```

### CODE REVIEW

**Comment by Akasurde** [comment]
cc @abadger 

Let me know if you need anything else.

File: lib/ansible/modules/cloud/amazon/cloudformation.py [hunk 1]
```diff
@@ -1,18 +1,9 @@
 #!/usr/bin/python
 # This file is part of Ansible
```
Comment (abadger) [inline lib/ansible/modules/cloud/amazon/cloudformation.py]: I like to get rid of this line (it's part of the long-form GPL recommendation and we're switching to the one-liner).

I also like to add a Copyright notice.  That can require trawling through history which can be time consuming so I have been doing:
``` python
# Copyright: (c) Ansible Project
```

Which is okay for our project.  (If anyone needs to do something legal-related, like relicense, we would have to trawl through git history no matter what the Copyright header says.)  It leaves out the copyright year which I dislike doing but the git history for modules can be time consuming to look through to find the last "copyrightable" change.  I figure if someone complains we can trawl through git history at that time.

For this particular module, we're lucky.  in git log -p I see that there were some copyrightable changes this year so the copyright line can be:
``` python
# Copyright: (c) 2017, Ansible Project
```

Comment (Akasurde) [reply lib/ansible/modules/cloud/amazon/cloudformation.py]: Sound good to me. I will do required changes.

**Comment by abadger** [comment]
Merged to devel.  Thanks @Akasurde !

**Comment by Akasurde** [comment]
@abadger Thanks for review and merge.

**Pull request merged**